### PR TITLE
Nathan Wick Code Review 2

### PR DESCRIPTION
**Pull request by nathan-wick**
Plan2Meet is an Android app that helps groups of people find a time to meet.

The program was available on UC Github on time.

The program is documented/commented on well enough for me to understand.

The program does compile.

The rationale behind my changes is each commit’s description.

Things that I learned during this sprint:

1. Android Studio can warn you about typos
2. How to implement a navigation drawer
3. How to use CircleCI

Commits I made to my evaluatee’s project:

1. https://github.com/WillTaylor2352/Plan2Meet/commit/b2f25ace785627e773313089a92df92e5cdeb673
2. https://github.com/WillTaylor2352/Plan2Meet/commit/2bf4e785f7c67c9b5f1d3b568bc4220e277e9a68
3. https://github.com/WillTaylor2352/Plan2Meet/commit/93a3b362ddc080a418b7b522ae6a0ad1d97b8569

Commits I made to my group’s project:

1. https://github.com/nathan-wick/IT3048C-FinalProject/commit/f9206caef9f993666679ecbc70a59f833751cb57
2. https://github.com/nathan-wick/IT3048C-FinalProject/commit/a7f3a3e4954e24930cf20f33e5a46000b15a0fa2
3. https://github.com/nathan-wick/IT3048C-FinalProject/commit/326ec9ab2e67be6760196f185660fa4cade99145

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -158,7 +158,7 @@
         android:layout_marginTop="484dp"
         android:layout_marginEnd="141dp"
         android:layout_marginBottom="109dp"
-        android:text="@string/set_availaiblity_text"
+        android:text="@string/set_availability_text"
```
Comment (discospiff) [inline app/src/main/res/layout/activity_main.xml:161]: Good catch.

File: app/src/main/res/values/themes.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
         <item name="colorSecondaryVariant">@color/teal_700</item>
         <item name="colorOnSecondary">@color/black</item>
         <!-- Status bar color. -->
-        <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>
+        <item name="android:statusBarColor">?attr/colorPrimaryVariant</item>
```
Comment (discospiff) [inline app/src/main/res/values/themes.xml:13]: What's the rationale for this change?

**Comment by discospiff** [comment]
From this review, I'd harvest the spelling error correction, const, and update to Retrofit package.

**Comment by WillTaylor2352** [comment]
As of right now, we will be proceeding with development without using the changes proposed in this pull request
